examples/dummy_plugin/dummy_plugin/homepage.py

```diff
@@ -0,0 +1,51 @@
+from nautobot.core.apps import HomePageItem, HomePagePanel
+
+from .models import DummyModel
+
+
+def get_dummy_data(request):
+    examples = DummyModel.objects.all()
+    if not examples:
+        examples = (
+            DummyModel.objects.create(name="Example 1", number=100),
+            DummyModel.objects.create(name="Example 2", number=200),
+            DummyModel.objects.create(name="Example 3", number=300),
+        )
+    return examples
+
+
+layout = (
+    HomePagePanel(
+        name="Organization",
+        items=(
+            HomePageItem(
+                name="Dummy Models",
+                model=DummyModel,
+                weight=150,
+                link="plugins:dummy_plugin:dummymodel_list",
+                description="List dummy plugin models.",
+                permissions=["dummy_plugin.view_dummymodel"],
+            ),
+        ),
+    ),
+    HomePagePanel(
+        name="Dummy Plugin",
+        weight=250,
+        items=(
+            HomePageItem(
+                name="Custom Dummy Plugin",
+                custom_template="panel_dummy_example.html",
```

Comment:
Does this actually render correctly? It seems weird to me to be using the same template for a `HomePagePanel` and a `HomePageItem` - I'd expect that you'd need a different template for one versus the other in order to appear correctly in the home page.

Reply:
This does render but does look different as the template is wrapped in a div with `list-group-item`.

Reply:
So, given that the dummy plugin is supposed to provide an example to follow, should we provide a different template for this case? I would think in general that the template for a HomePagePanel would be different than the template for a HomePageItem.